geomesa-accumulo/geomesa-accumulo-dist/pom.xml

```diff
@@ -37,10 +37,6 @@
             <groupId>org.locationtech.geomesa</groupId>
             <artifactId>geomesa-spark-converter_${scala.binary.version}</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-spark-geotools_${scala.binary.version}</artifactId>
```

Comment:
Was there any implicit dependencies inside 'geomesa-spark-geotools' that GM+Accumulo+Spark needed?

Reply:
I'm fairly sure not - I don't remember if I tested accumulo or not, but we cover testing spark during our release process, so we should catch any problems then.